cobbler/actions/buildiso/netboot_buildiso.py

```diff
@@ -0,0 +1,606 @@
+"""
+This module contains the specific code to generate a network bootable ISO.
+"""
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import os
+import re
+from typing import List
+
+from cobbler import utils
+from cobbler.actions import buildiso
+
+
+def _my_int_append_line(my_int, data: dict, dist, append_line: str) -> str:
+    """
+    This generates the interface configuration for the system to boot for the append line.
+
+    :param my_int: The interface where the information should be assigned to.
+    :param data: The data which the append line is generated from.
+    :param dist: The distribution to generate the append line for.
+    :param append_line: The previously existing append line.
+    :return: The append line with the interface configuration included.
+    """
+    if my_int is not None:
+        intmac = "mac_address_" + my_int
+        if dist.breed == "suse":
+            if intmac in data and data[intmac] != "":
+                append_line += " netdevice=%s" % data["mac_address_" + my_int].lower()
+            else:
+                append_line += " netdevice=%s" % my_int
+        if dist.breed == "redhat":
```

Comment:
I don't like these separate conditions for dist.breed. I think it's better to replace with `if ... elif ...`. In case of having separate `if`s each of them is evaluated, it's not really significant in performance point of view, but maybe try to get rid of separate conditional blocks?

Reply:
Should be fixed with a7f3add